usr/share/rear/verify/GALAXY11/default/500_select_backupset.sh

```diff
@@ -7,5 +7,5 @@ IFS=$'\n' read -r -d "" -a backupsets < <(
 )
 
 until IsInArray "$GALAXY11_BACKUPSET" "${backupsets[@]}"; do
-	GALAXY11_BACKUPSET=$( UserInput -I GALAXY11_BACKUPSET -p "Select CommVault backupset to use:" "${backupsets[@]}" )
+	GALAXY11_BACKUPSET=$( UserInput -I GALAXY11_BACKUPSET -D "${backupsets[0]}" -p "Select CommVault backupset to use (ENTER for first one):" "${backupsets[@]}" )
```

Comment:
@codefritzel is it OK to use the first backup set as default choice?

Reply:
Its OK.
I thought we had already added that.